src/components/ContentPreview/ContentPreview.js

```diff
@@ -360,6 +362,7 @@ class ContentPreview extends PureComponent<Props, State> {
             header: 'none',
             skipServerUpdate: true,
             useHotkeys: false,
+            showDownload: canDownload,
```

Comment:
@priyajeet print and download are coupled by the same option (showDownload) in preview, so passing in a separate option for print is not needed.

Reply:
Need to guard the download() and print() functions below too.
And then pass canDownload to the header to not show the buttons.